examples/07_diagnostic_plot.py

```diff
@@ -0,0 +1,15 @@
+"""
+Diagnostic plot
+-------------------
+
+A diagnostic plot is a simultaneous plot of the drawdown and the
+logarithmic derivative of the drawdown in a log-log plot.
+Often, this plot is used to identify the right approach for the aquifer estimations.
+
+"""
+
+
+import welltestpy as wtp
+
+campaign = wtp.load_campaign("Cmp_UFZ-campaign.cmp")
+campaign.diagnostic_plot("well_0", "well_1")
```

Comment:
Ahh. You need to add an argument `rate` to `diagnostic_plot_pump_test` which is the pumping rate. When `rate<0` you have to multiply head by `-1` to get the "drawdown" and not the "piezometric level". Then, in the method `PumpingTest.diagnostic_plot` you need to pass `self.pumpingrate`.